imgProcess.py

```diff
@@ -0,0 +1,72 @@
+# Tayo Elelu
+# Created 10/12/16
+# CS 4365 - Software Engr II
+# Image Classification Module
+
+# This module is used to generalize image classification
+# for the project. Ultimately, it should be able to take in a
+# group of images and classify them to a specified target, and print results.
+
+# 1 - Group of Images to CSV file Script
+# 2 - Apply svm.svc to CSV of images
+
+# PROBLEM - to use Scikit for image processing, number of pixels
+# must be the same in each image, so padding/resizing/reshaping is
+# needed. Drastically limits the generality of a module.
+
+##############################################################
+
+# Standard scientific Python imports
+import matplotlib
+matplotlib.use('Agg')         #Force matplotlib to not use any Xwindows backend (causes errors)
+import matplotlib.pyplot as plt
+
+# Import datasets, classifiers and performance metrics
+from sklearn import datasets, svm, metrics
+
+# The digits dataset
+digits = datasets.load_digits()
+
+# The data that we are interested in is made of 8x8 images of digits, let's
+# have a look at the first 4 images, stored in the `images` attribute of the
+# dataset.  If we were working from image files, we could load them using
+# matplotlib.pyplot.imread.  Note that each image must have the same size. For these
+# images, we know which digit they represent: it is given in the 'target' of
+# the dataset.
+images_and_labels = list(zip(digits.images, digits.target))
+for index, (image, label) in enumerate(images_and_labels[:4]):
+    plt.subplot(2, 4, index + 1)
+    plt.axis('off')
+    plt.imshow(image, cmap=plt.cm.gray_r, interpolation='nearest')
+    plt.title('Training: %i' % label)
+
+# To apply a classifier on this data, we need to flatten the image, to
+# turn the data in a (samples, feature) matrix:
+n_samples = len(digits.images)
+data = digits.images.reshape((n_samples, -1))
+
+# Create a classifier: a support vector classifier
+classifier = svm.SVC(gamma=0.001)

```

Comment:
This shouldn't be done here, if you need a new ML algorithm please add it to our model management system.